media/css/firefox/firstrun/firstrun-quantum.scss

```diff
@@ -35,12 +39,9 @@ body {
     background: url("/media/img/firefox/firstrun/fox-tail-header.png") top -200px center no-repeat,
                 linear-gradient(to bottom, #003eaa 40%, #004ec2 60%, #0060df 80%, #0080ff 90%, #00c7ff 100%) top center no-repeat,
                 #003eaa;
-    margin: 0;
-    min-height: 100vh;
-}
-
-* {
     font-family: 'Fira Sans Light', sans-serif;
+    min-height: 100vh;
+    -moz-osx-font-smoothing: unset;
```

Comment:
This bit was needed because the global font smoothing from Protocol really doesn't play well with Fira Sans Light on this background. Compared to the Pebbles-based page the text was extra thin. I wonder if we should remove the font smoothing in Protocol... but then again, it's really only a problem with light fonts so I think this hack will work for now on this one page (which needs a rebranding soon anyway).